CONTRIBUTING.md

```diff
@@ -72,71 +92,79 @@ By making a contribution to this project, I certify that:
+**Pre-Release Version**
+In-progress release
+- :crystal_ball: `:crystal_ball:`
 
-New functionality always exists in relation to a specific type of user. When requesting new functionality, include the user persona the functionality relates to, the need, and the version of the codebase. Titles should be written with the user persona first, then the a short description of their need (5-10 words).
+The version that gets generated is based on the highest type of change identified; i.e. if there were five _patch_ commits and one _minor_ commit, the _minor_ commit takes precedence and a new _minor_ release will be generated. All contributing commits will be included in the changelog.
 
-New functionality also needs their requirements for being considered complete written out. To do so, features and scenarios should be written to describe the expected behavior in [Gherkin](https://github.com/cucumber/cucumber/wiki/Gherkin) syntax. Each feature and scenario par should have a [task list](https://github.com/blog/1375%0A-task-lists-in-gfm-issues-pulls-comments) created to track their completion. The issue is considered **done** when all feature/scenario pairs defined have been completed with passing tests written.
+## Issue Guidelines
 
-The following can be copied as a templates to follow:
+There are generally two kinds of issues: functionality requests and bugs. Each is treated slightly differently, and there is a template for each.
 
-**Title**
+#### Functionality Requests
 
-    {{persona}}: {{need}}
+[Request New Functionality](https://github.com/punchcard-cms/punchcard/issues/new)
 
-**Body**
+New functionality always exists in relation to a specific type of user. When requesting new functionality, include the user persona the functionality relates to, the need for that functionality, and the rationale as to why this is needed. Titles should be written with the user persona first, then the a short description of their need (5-10 words).
 
-    {{long description}}
-    
-    ```gherkin
-    Feature: {{feature}}
-      As a {{persona}}
-      I want {{need}}
-      So that {{rationale}}
-    
-      Scenario: {{scenario}}
-        Given {{thing}}
-        When {{action}}
-        Then {{result}}
-    ```
-    
-    ---
-    
-    - [ ] {{feature}}/{{scenario}}
+New functionality also needs their requirements for being considered complete written out. To do so, features and scenarios should be written to describe the expected behavior in [Gherkin](https://github.com/cucumber/cucumber/wiki/Gherkin) syntax. Each feature and scenario par should have a [task list](https://github.com/blog/1375%0A-task-lists-in-gfm-issues-pulls-comments) created to track their completion. The issue is considered **done** when all scenarios defined have been completed with passing tests written.
```

Comment:
spelling: `scenario pair should`
rewrite? `The issue is considered **done** only when _all_ scenarios defined have been completed, code is covered with testing, and all tests pass.`